CMakeLists.txt

```diff
@@ -78,20 +78,35 @@ get_filename_component(ZEEK_SCRIPT_INSTALL_PATH ${ZEEK_SCRIPT_INSTALL_PATH}
 
 set(BRO_PLUGIN_INSTALL_PATH ${ZEEK_ROOT_DIR}/lib/zeek/plugins CACHE STRING "Installation path for plugins" FORCE)
 
+set(bro_plugin_install_path "${BRO_PLUGIN_INSTALL_PATH}")
```

Comment:
I just noticed this after making my comment on zeek-config.in. So these are paths that have been cleared of any windows volume identifiers?

Reply:
Yes, though the volume identifiers have been MSYS-ified (by converting c:/ to /c/) rather than replaced.